src/frequenz/sdk/microgrid/grid.py

```diff
@@ -0,0 +1,82 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Grid connection point.
+
+This module provides the `Grid` type, which represents a grid connection point
+in a microgrid.
+"""
+
+import logging
+from dataclasses import dataclass
+from typing import Iterable, Optional
+
+from .component import Component
+from .component._component import ComponentCategory
+from .fuse import Fuse
+
+
+@dataclass(frozen=True)
+class Grid:
+    """A grid connection point.
+
+    Attributes:
+        fuse: The fuse protecting the grid connection point.
+    """
+
+    fuse: Fuse
+
+
+_GRID: Optional[Grid] = None
+
+
+def initialize(components: Iterable[Component]) -> None:
+    """Initialize the grid connection.
+
+    Args:
+        components: The components in the microgrid.
+
+    Raises:
+        RuntimeError: If there is more than 1 grid connection point in the
+            microgrid.
```

Comment:
Actually there are 3 cases where `RuntimeError` is raised that you might want briefly to mention them

Reply:
Done